src/systems/userinput/bindings/oculus-touch-user.js

```diff
@@ -843,36 +931,70 @@ export const oculusTouchUserBindings = addSetsToBindings({
     },
     {
       src: [cursorDrop1, cursorDrop2],
-      dest: { value: paths.actions.cursor.drop },
+      dest: { value: paths.actions.cursor.right.drop },
       xform: xforms.any,
       priority: 2
+    }
+  ],
+
+  [sets.leftCursorHoldingInteractable]: [
```

Comment:
Not really something to fix in this PR, but definitely don't love the level of "almost exactly the same" code we have here for left/right handed stuff. Definitely one of the things to address when we make another pass on input.